charts/_templates.gotmpl

```diff
@@ -25,6 +29,8 @@ products using R and Python.
 # RStudio Package Manager
   {{- else if eq .Name "rstudio-workbench" -}}
 # RStudio Workbench
+  {{- else if eq .Name "posit-chronicle" -}}
+# Posit Chronicle
```

Comment:
Yeah, it is a tough call. Either way, it is a bit awkward. I think I would vote for calling it `posit-chronicle` as you have since that it the actual product name. Hopefully, in the future, we can update the other charts in a non-breaking way.